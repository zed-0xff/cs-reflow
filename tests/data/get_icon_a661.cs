	public unsafe static string get_icon_a661(object object_0a)
	{
		string text = get_str(72129, 73044, 169);
		int num2 = 0;
		int num4;
		int QRR = Structs_e0d5.QRR;
		uint num12 = default(uint);
		int num13 = default(int);
		int tRNS = default(int);
		uint num16 = default(uint);
		int num10 = default(int);
		int num5 = default(int);
		int num19 = default(int);
		int num20 = default(int);
		do
		{
			int num3 = num2 = 19;
            num4 = num5 - 7686 << 7;
			string text2 = get_str(8975, 9889, 160);
			int num6 = 23;
			string text3 = get_str(44130, 45101, 188);
			int num8 = 0;
			while (true)
			{
				int num9 = num8 = 20;
				bool ab759f63c28b4fceadd28e63ce8bde = Class_ae29.flag_ab75;
				if (!ab759f63c28b4fceadd28e63ce8bde)
				{
					goto IL_02ef;
				}
				goto IL_0449;
				IL_0a93:
				return string.Empty;

				IL_0490:
				int num11 = -385269278;
				goto IL_036c;

				IL_01fc:
				switch ((num12 = (uint)(-(~(-(-((num13 - 1810079697) * -1505781987) - -1731946980) - -1744120490)) ^ 0xFE2E671)) % 33) // 29
				{
				case 13u:
					break;
				case 32u:
					goto IL_02b4;
				case 26u:
					goto IL_0332;
				case 21u:
					goto IL_0449;
				case 28u:
					goto IL_0490;
				default:
					goto IL_049a;
				case 16u:
					return Class_ae29.b64_axisPay_jpg;
				case 29u:
					goto IL_04b3;
				case 0u:
					goto IL_0501;
				case 31u:
					goto IL_052a;
				case 18u:
					goto IL_0572;
				case 12u:
					goto IL_05a0;
				case 8u:
					goto IL_05e0;
				case 11u:
					goto IL_0629;
				case 14u:
					goto IL_0642;
				case 17u:
					goto IL_0687;
				case 1u:
					goto IL_06b0;
				case 19u:
					goto IL_0703;
				case 25u:
					goto IL_072b;
				case 24u:
					goto IL_0754;
				case 22u:
					goto IL_0780;
				case 15u:
					goto IL_07c7;
				case 23u:
					goto IL_0809;
				case 30u:
					goto IL_0850;
				case 9u:
					goto IL_088a;
				case 10u:
					return Class_ae29.b46_secureOTP_jpg;
				case 20u:
					goto IL_08f2;
				case 7u:
					return Class_ae29.a715a55768504143ad6674958663538e;
				case 4u:
					goto IL_093c;
				case 5u:
					goto IL_0985;
				case 27u:
					goto IL_09ef;
				case 2u:
					goto IL_0a31;
				case 3u:
					goto IL_0a5d;
				case 6u:
					goto IL_0a93;
				}
				break;
				IL_0a5d:
				int num14;
				if (String.op_Equality((string)object_0a, "com.sbi.upi"))
				{
					num11 = 55004113;
					num14 = 55004113;
					goto IL_036c;
				}
				goto IL_08c9;
				IL_0572:
				Class_ae29.b64_a16d = "AAEAAAAOAIAAAwBgT1MvMm+hhd4AAADsAAAAVmNtYXBA5XCVAA";
				num11 = ((int)num12 * -212542682) ^ 0x2AA796A8;
				goto IL_036c;
				IL_0a31:
				Class_ae29.a715a55768504143ad6674958663538e = "/9j/4AAQSkZJRgABAQEAAAAAAAD/4QAqRXhpZgAATU0AKgAAAA";
				num11 = ((int)num12 * -86184403) ^ 0x3BB781B8;
				goto IL_036c;
				IL_09ef:
				Class_ae29.phones_dict.Add("Samsung Galaxy M34 5G", "Samsung Galaxy M34 5G|Samsung|M346B|6.5|8GB|Android|13");
				num11 = ((int)num12 * -1299170994) ^ 0x7DCD3C8C;
				goto IL_036c;
				IL_0985:
				Class_ae29.phones_dict.Add("Samsung Galaxy A15 5G", "Samsung Galaxy A15 5G|Samsung|A156E|6.5|8GB|Android|14");
				num11 = (int)(num12 * 1602542612) ^ -1918859128;
				goto IL_036c;
				IL_04b3:
				Dictionary<string, string> phones_dict_ = Class_ae29.phones_dict;
				string key = "Samsung Galaxy A54 5G";
				tRNS = Structs_e0d5.tRNS;
				phones_dict_.Add(key, "Samsung Galaxy A54 5G|Samsung|A546|6.4|8GB|Android|13");
				num11 = ((int)num12 * -130994618) ^ 0x1B87E56;
				goto IL_036c;

				IL_049a:
				int num15 = ((int)num16 * -1476652062) ^ 0x3008CA33;
				goto IL_0301;

				IL_0501:
				Class_ae29.b64_a6c1 = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QBaRXhpZgAATU0AKgAAAA";
				num11 = (int)(num12 * 32601071) ^ -352244026;
				goto IL_036c;

				IL_052a:
				Class_ae29.phones_dict.Add("Samsung Galaxy Flip5 5G", "Samsung Galaxy Flip5 5G|Samsung|F731B|6.5|8GB|Android|13");
				num10 = a73606a184b74efcaff0b2b5ed355f93;
				num11 = ((int)num12 * -201740808) ^ -1080280616;
				goto IL_036c;

				IL_093c:
				Class_ae29.phones_dict.Add("Samsung Galaxy A35 5G", "Samsung Galaxy A35 5G|Samsung|A356E|6.6|8GB|Android|14");
				num5 = a140f784cad84b15b4faa8c35c20381e;
				num11 = (int)((num12 * 337529798) ^ 0x4DF25C28);
				goto IL_036c;

				IL_08f2:
				int num17;
				if (String.op_Equality((string)object_0a, "com.upi.axispay"))
				{
					num11 = -1300579427;
					num17 = -1300579427;
				}
				else
				{
					num11 = 64097946;
					num17 = 64097946;
				}
				goto IL_036c;

				IL_0449:
				int num18;
				if (String.op_Equality((string)object_0a, "in.org.npci.upiapp"))
				{
					num11 = 567590242;
					num18 = 567590242;
				}
				else
				{
					num11 = -926362356;
					num18 = -926362356;
				}
				goto IL_036c;

				IL_0332:
				Class_ae29.phones_dict.Add("Samsung Galaxy F54 5G", "Samsung Galaxy F54 5G|Samsung|E546BZSHINS|6.7|8GB|Android|13");
				num11 = (int)((num12 * 1273216926) ^ 0x3D025C9B);
				goto IL_036c;

				IL_088a:
				Class_ae29.phones_dict.Add("Samsung Galaxy M14 5G", "Samsung Galaxy M14 5G|Samsung|M146B|6.6|8GB|Android|13");
				if ((int)((uint)(num2 >>> 4) & (num12 / 65536)) >= 0)
				{
					num11 = ((int)num12 * -2127121915) ^ -947815158;
					goto IL_036c;
				}
				goto IL_08c9;
				IL_08c9:
				num11 = -1221962382;
				num14 = -1221962382;
				goto IL_036c;
				IL_02ef:
				num15 = 1493998119;
				goto IL_0301;
				IL_0850:
				Class_ae29.b64_axisPay_jpg = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QCSRXhpZgAATU0AKgAAAA";
				num11 = (int)(num12 * 27053829) ^ -538205153;
				goto IL_036c;
				IL_0809:
				Class_ae29.phones_dict.Add("OnePlus 7TPro", "OnePlus 7TPro|OnePlus|HD1911|6.6|8GB|Android|11");
				num11 = (int)(num12 * 646987686) ^ -985347252;
				goto IL_036c;
				IL_07c7:
				Class_ae29.phones_dict.Add("Samsung Galaxy S24 Ultra 5G", "Samsung Galaxy S24 Ultra 5G|Samsung|S928BZ|6.8|8GB|Android|14");
				num11 = (int)((num12 * 1791021123) ^ 0x4482F6AF);
				goto IL_036c;
				IL_0780:
				Class_ae29.phones_dict.Add("Samsung M04", "Samsung M04|Samsung|M045F|6.5|8GB|Android|12");
				num11 = (int)(num12 * 309523672) ^ -1136935488;
				goto IL_036c;
				IL_0754:
				Class_ae29.b64_ac6a = "iVBORw0KGgoAAAANSUhEUgAAABYAAAAWCAYAAADEtGw7AAAABG";
				num11 = (int)(num12 * 2014062606) ^ -1080775199;
				goto IL_036c;
				IL_072b:
				Class_ae29.b64_fae3 = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QBaRXhpZgAATU0AKgAAAA";
				num11 = ((int)num12 * -1018750194) ^ 0x38451CEA;
				goto IL_036c;
				IL_0703:
				Class_ae29.b64_ac6a = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QBaRXhpZgAATU0AKgAAAA";
				num11 = (int)(num12 * 1694902059) ^ -933540649;
				goto IL_036c;

				IL_06b0:
				Class_ae29.b46_secureOTP_jpg = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QCSRXhpZgAATU0AKgAAAA";
				num11 = ((int)num12 * -1786658800) ^ 0x6FE95EC7;
				goto IL_036c;
				IL_036c:
				num13 = num11;
				num15 = 1447283596;
				goto IL_0301;
				IL_0301:
				uint num21 = (num16 = (uint)(num15 ^ 0x5F65DA59)) % 5;
				switch (num21) // 2
				{
				case 2u:
					break;
				case 0u:
					goto IL_02ef;
				default:
					goto IL_0332;
				case 1u:
					goto IL_0490;
				case 3u:
					goto IL_0a93;
				}
				goto IL_01fc;
				IL_02b4:
				Class_ae29.phones_dict.Add("Samsung Galaxy S20 FE 5G", "Samsung Galaxy S20 FE 5G|Samsung|G781BZBGINS|6.5|8GB|Android|11");
				num11 = ((int)num12 * -1338194159) ^ -2114570542;
				goto IL_036c;
				IL_0687:
				Class_ae29.ae97e074e9e7420fa7898158aca772e9 = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QCyRXhpZgAATU0AKgAAAA";
				num11 = ((int)num12 * -1858635000) ^ -1420093862;
				goto IL_036c;
				IL_0642:
				Class_ae29.phones_dict.Add("Samsung Galaxy S22 5G", "Samsung Galaxy S22 5G|Samsung|S901EZGDINU|6.1|8GB|Android|12");
				num11 = (int)((num12 * 787453547) ^ 0x78DDE905);
				goto IL_036c;
				IL_0629:
				Class_ae29.flag_ab75 = true;
				num11 = ((int)num12 * -867220638) ^ 0x47E89ABD;
				goto IL_036c;
				IL_05e0:
				Dictionary<string, string> phones_dict_2 = Class_ae29.phones_dict;
				string key2 = "Samsung Galaxy A23 5G";
				num19 = a999fbf5299144caa731c75edd1c4be0;
				phones_dict_2.Add(key2, "Samsung Galaxy A23 5G|Samsung|A236E|6.6|8GB|Android|12");
				num11 = ((int)num12 * -14019062) ^ 0x1BE0FB73;
				goto IL_036c;
				IL_05a0:
				int num22;
				if (!String.op_Equality((string)object_0a, "com.sbi.SBISecure"))
				{
					num11 = 1334928138;
					num22 = 1334928138;
				}
				else
				{
					num11 = -940845874;
					num22 = -940845874;
				}
				goto IL_036c;
			}
			break;
		}
		while ((1 & (((num4 | (QRR << 26)) >>> 7) ^ (num5 + 2028))) == 0);
		return Class_ae29.ae97e074e9e7420fa7898158aca772e9;
	}
