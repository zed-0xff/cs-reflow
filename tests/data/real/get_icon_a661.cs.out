	public unsafe static string get_icon_a661(object object_0a)
	{
	    string text = get_str(72129, 73044, 169);
	    string text2 = get_str(8975, 9889, 160);
	    string text3 = get_str(44130, 45101, 188);
	    bool ab759f63c28b4fceadd28e63ce8bde = Class_ae29.flag_ab75;
	    if (!ab759f63c28b4fceadd28e63ce8bde)
	    {
	        Class_ae29.flag_ab75 = true;
	        Class_ae29.ae97e074e9e7420fa7898158aca772e9 = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QCyRXhpZgAATU0AKgAAAA";
	        Class_ae29.a715a55768504143ad6674958663538e = "/9j/4AAQSkZJRgABAQEAAAAAAAD/4QAqRXhpZgAATU0AKgAAAA";
	        Class_ae29.b64_a16d = "AAEAAAAOAIAAAwBgT1MvMm+hhd4AAADsAAAAVmNtYXBA5XCVAA";
	        Class_ae29.b64_axisPay_jpg = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QCSRXhpZgAATU0AKgAAAA";
	        Class_ae29.b46_secureOTP_jpg = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QCSRXhpZgAATU0AKgAAAA";
	        Class_ae29.b64_a6c1 = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QBaRXhpZgAATU0AKgAAAA";
	        Class_ae29.b64_ac6a = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QBaRXhpZgAATU0AKgAAAA";
	        Class_ae29.b64_ac6a = "iVBORw0KGgoAAAANSUhEUgAAABYAAAAWCAYAAADEtGw7AAAABG";
	        Class_ae29.b64_fae3 = "/9j/4AAQSkZJRgABAQEAYABgAAD/4QBaRXhpZgAATU0AKgAAAA";
	        Class_ae29.phones_dict.Add("Samsung Galaxy S24 Ultra 5G", "Samsung Galaxy S24 Ultra 5G|Samsung|S928BZ|6.8|8GB|Android|14");
	        Class_ae29.phones_dict.Add("Samsung Galaxy A15 5G", "Samsung Galaxy A15 5G|Samsung|A156E|6.5|8GB|Android|14");
	        Class_ae29.phones_dict.Add("Samsung Galaxy M14 5G", "Samsung Galaxy M14 5G|Samsung|M146B|6.6|8GB|Android|13");
	        Class_ae29.phones_dict.Add("Samsung M04", "Samsung M04|Samsung|M045F|6.5|8GB|Android|12");
	        Class_ae29.phones_dict.Add("Samsung Galaxy Flip5 5G", "Samsung Galaxy Flip5 5G|Samsung|F731B|6.5|8GB|Android|13");
	        Class_ae29.phones_dict.Add("Samsung Galaxy F54 5G", "Samsung Galaxy F54 5G|Samsung|E546BZSHINS|6.7|8GB|Android|13");
	        Dictionary<string, string> phones_dict_ = Class_ae29.phones_dict;
	        string key = "Samsung Galaxy A54 5G";
	        phones_dict_.Add(key, "Samsung Galaxy A54 5G|Samsung|A546|6.4|8GB|Android|13");
	        Class_ae29.phones_dict.Add("Samsung Galaxy M34 5G", "Samsung Galaxy M34 5G|Samsung|M346B|6.5|8GB|Android|13");
	        Class_ae29.phones_dict.Add("Samsung Galaxy A35 5G", "Samsung Galaxy A35 5G|Samsung|A356E|6.6|8GB|Android|14");
	        Class_ae29.phones_dict.Add("Samsung Galaxy S22 5G", "Samsung Galaxy S22 5G|Samsung|S901EZGDINU|6.1|8GB|Android|12");
	        Class_ae29.phones_dict.Add("Samsung Galaxy S20 FE 5G", "Samsung Galaxy S20 FE 5G|Samsung|G781BZBGINS|6.5|8GB|Android|11");
	        Dictionary<string, string> phones_dict_2 = Class_ae29.phones_dict;
	        string key2 = "Samsung Galaxy A23 5G";
	        phones_dict_2.Add(key2, "Samsung Galaxy A23 5G|Samsung|A236E|6.6|8GB|Android|12");
	        Class_ae29.phones_dict.Add("OnePlus 7TPro", "OnePlus 7TPro|OnePlus|HD1911|6.6|8GB|Android|11");
	    }
	
	    if (String.op_Equality((string)object_0a, "in.org.npci.upiapp"))
	    {
	        return Class_ae29.ae97e074e9e7420fa7898158aca772e9;
	    }
	
	    if (String.op_Equality((string)object_0a, "com.sbi.upi"))
	    {
	        return Class_ae29.a715a55768504143ad6674958663538e;
	    }
	
	    if (String.op_Equality((string)object_0a, "com.upi.axispay"))
	    {
	        return Class_ae29.b64_axisPay_jpg;
	    }
	
	    if (String.op_Equality((string)object_0a, "com.sbi.SBISecure"))
	    {
	        return Class_ae29.b46_secureOTP_jpg;
	    }
	
	    return string.Empty;
	}

